metis-core/metis-core-common/src/main/java/eu/europeana/metis/core/workflow/plugins/ThrottlingValues.java

```diff
@@ -0,0 +1,68 @@
+package eu.europeana.metis.core.workflow.plugins;
+
+/**
+ * Class encapsulating all possible throttling levels tuples
+ *
+ * @author Joana Sousa (joana.sousa@europeana.eu)
+ * @since 2022-08-12
+ */
+public class ThrottlingValues {
+
+    private final ThrottlingLevelValuePair weak;
+    private final ThrottlingLevelValuePair medium;
+    private final ThrottlingLevelValuePair strong;
```

Comment:
The reason why I created a class only for the pair was mainly for readability purposes, I wanted to make it clearer what I am getting instead of `.getLeft()` or `.getRight()`. The enum `ThrottlingLevel` was created so I could limit the number of possible values for levels, I didn't want to give space for possible errors.
But this is a very good point, I didn't notice I could have done it simpler. I will remove the pair and enum class and focus on this class mainly

Reply:
I think the Enum class should have stayed. Now you have some string constants, which I think is not an improvement, don't you think? Perhaps I was not being clear... I think the pairs were unnecessary, not the enum value.